game/scripts/npc/abilities/chen_divine_favor.txt

```diff
@@ -8,15 +8,28 @@
 		// General
 		//-------------------------------------------------------------------------------------------------------------
 		"ID"							"7306"														// unique ID number for this ability.  Do not change this once established or it will invalidate collected stats.
-		"AbilityBehavior"				"DOTA_ABILITY_BEHAVIOR_PASSIVE"
+		"AbilityBehavior"				"DOTA_ABILITY_BEHAVIOR_UNIT_TARGET"
```

Comment:
you know, you only need to put in values that actually change. By adding these extra things in these files you just kinda create more work

Reply:
So I could just remove the line of ability behavior?.  In the Dota KV file "DOTA_ABILITY_BEHAVIOR_PASSIVE" was replaced because his teleport got put on divine foavour. So its a passive aura with an active component now.

Reply:
well the previous value had changed so it needed to either be changed or removed, but you also added 4 or 5 more lines, that's more what i was talking about.